src/function/pragma/pragma_queries.cpp

```diff
@@ -22,21 +22,27 @@ string PragmaShowTables(ClientContext &context, const FunctionParameters &parame
 	auto schema = ClientData::Get(context).catalog_search_path->GetDefault().schema;
 	schema = (schema == INVALID_SCHEMA) ? DEFAULT_SCHEMA : schema; // NOLINT
 
-	auto where_clause =
-	    StringUtil::Join({"where database_name = '", catalog, "' and schema_name = '", schema, "'"}, "");
+	auto where_clause = StringUtil::Format("where ((database_name = '%s') and (schema_name = '%s'))", catalog, schema);
+	// clang-format off
+	auto pragma_query = StringUtil::Format("\
+	with \"tables\" as						\
```

Comment:
Can we use a multi-line string literal here? e.g.:

```cpp
auto pragma_query = R"(
      ...
    )";
```

Reply:
Learned something new today, that's really cool! No need to escape the newlines, and also no need to escape the `"` 🤯